lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -777,7 +777,17 @@ module Resources
             'start': '2012-05-17T15:33:18.000Z'
           }
         }
-      }
+      },
+      {
+        'op': 'test',
+        'path': '/name/0/id',
+        'value': 'CI-12724068-12724066-0'
```

Comment:
Gonna have o put these back, but make sure it's a valid name id in C1941. Can you also fix the name id examples in else where in this file, for example: https://github.com/cerner/fhir.cerner.com/blob/main/lib/resources/example_json/r4_examples_relatedperson.rb#L91

Reply:
Changed the value here and other places from your prod validation jira, OPENSVC-15683. Is this fine?
Is there any other place I am missing, where the id needs to be replaced?

https://github.com/cerner/fhir.cerner.com/commit/c9bce63c67bb7f03d1c33b644945dfaa15d064db#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR784